### PR TITLE
Add the Create_bbox_mesh_plugin

### PR DESCRIPTION
**Pull request by lrineau**
Add a new plugin to compute bounding polyhedra.

Maybe the trivial plugin should be removed.

@afabri @sloriot @janetournois

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Plugins/PCA/Create_bbox_mesh_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            << std::endl;
+
+  if(extended) {
+    const double delta_x = ( bbox.xmax - bbox.xmin ) / 20.;
+    const double delta_y = ( bbox.ymax - bbox.ymin ) / 20.;
+    const double delta_z = ( bbox.zmax - bbox.zmin ) / 20.;
+    // const double center_x = ( bbox.xmax + bbox.xmin ) / 2.;
+    // const double center_y = ( bbox.ymax + bbox.ymin ) / 2.;
+    // const double center_z = ( bbox.zmax + bbox.zmin ) / 2.;
+
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Plugins/PCA/Create_bbox_mesh_plugin.cpp]: Commented code should be removed ;)

File: Polyhedron/demo/Polyhedron/Plugins/PCA/Create_bbox_mesh_plugin.cpp [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  Polyhedron p;
+  
+  Build_bbox_mesh b(bbox);
+  p.delegate(b);
+
+  Scene_item* item = new Scene_polyhedron_item(p);
+  item->setName("Scene bbox mesh");
+  // item->setColor(Qt::black);
+  item->setRenderingMode(Wireframe);
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Plugins/PCA/Create_bbox_mesh_plugin.cpp]: same here

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-111

**Comment by sloriot** [comment]
@lrineau This PR can be merged into master, up to `cgal_test_with_cmake` that requires an update to handle this plugin.
The trivial plugin can be removed as it is duplicated with the package `Three`.

**Comment by lrineau** [comment]
As the documentation Three will be removed from CGAL-4.8, because it is not reviewed, I think we should keep the trivial plugin for a while.

**Comment by lrineau** [comment]
2692410 modifies `cgal_test_with_cmake`. I am surprised that it still works, after the move of plugins to sub-directories.

**Pull request merged**